profiles/keycloak_integration.py

```diff
@@ -8,7 +10,7 @@
 )
 from utils import keycloak
 
-_keycloak_admin_client = None
+_keycloak_admin_client: Optional[keycloak.KeycloakAdminClient] = None
```

Comment:
I guess it's a matter of taste, but since Python 3.10 you can also use:
```suggestion
_keycloak_admin_client: keycloak.KeycloakAdminClient | None = None
```

Reply:
Yeah, no, that's definitely how it should be done with Python >=3.10.